### PR TITLE
updated NBs plus code mods to get autocnet to work with ISIS4

### PR DESCRIPTION
**Pull request by ladoramkershner**

### CODE REVIEW

File: autocnet/matcher/subpixel.py [hunk 1]
```diff
@@ -1474,7 +1474,8 @@ def subpixel_register_point(pointid,
 
     version = version.lower()
     geom_funcs = {"classic": geom_match_classic,
-                "new": geom_match
+                "new": geom_match,
+                "simple": geom_match_simple
```
Comment (jessemapel) [inline autocnet/matcher/subpixel.py:1478]: Is this going to impact any current workflows? I'm assuming this is just an extra option.

**Review by jessemapel — APPROVED** [review]

File: autocnet/matcher/subpixel.py (hunk 1 above)
Comment (ladoramkershner) [reply autocnet/matcher/subpixel.py:1478]: It shouldn't, but the 'new' and 'classic' methods actually do not work at the moment

Comment (ladoramkershner) [reply autocnet/matcher/subpixel.py:1478]: Not relevant to this PR, but frustrating

**Comment by ladoramkershner** [comment]
It should be good for a rereview! The notebook is formatted for the workshop as well, blanks where blanks are needed and the autocnet_workshop kernel

**Review by jessemapel — APPROVED** [review]

**Pull request merged**